index.html

```diff
@@ -29,25 +29,112 @@
+                </div>
+                <div class="work-card">
+                    <div class="card-content">
+                        <h4>Professional Art Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="work-card">
+                    <div class="card-content">
+                        <h4>Professional Art Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
             </div>
-        </div>
-        <div>
-            <img class="landing-image " src="image/landig-image.jpg" alt="landing-image "> </div>
+        </section>
+        <section class="about-me">
+            <div class="about-text">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button">Get My Resume</button>
+                <hr class="hr">
+            </div>
+            <div class="skill-set">
+                <ul>
+                    <li class="languages"> Languages</li>
+                    <li class="skill-item">Javascript</li>
+                    <li class="skill-item">Ruby</li>
+                    <li class="skill-item">Html</li>
+                    <li class="skill-item">Css</li>
+                    <li class="frameworks"> Frameworks</li>
+                    <li class="skill-item">Bootstrap</li>
+                    <li class="skill-item">Ruby on Rails</li>
+                    <li class="skill-item">RSPec</li>
+                    <li class="skill-item">CapyBara</li>
+                    <li class="skill-item">Selenium</li>
+                    <li class="skills"> Skills</li>
+                    <li class="skill-item">Codekit</li>
+                    <li class="skill-item">Github</li>
+                    <li class="skill-item">Codepen</li>
+                    <li class="skill-item">Gitlab</li>
+                    <li class="skill-item">Terminal</li>
+                </ul>
```

Comment:
- [ ] Good job using `ul` and `li` tags here 💪. However, please consider putting the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, and each of them in a `li`. Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.  See suggestion below:

```suggestion
                <ul>
                  <li class="languages"> 
                      Languages
                       <ul>
                          <li class="skill-item">Javascript</li>
                          <li class="skill-item">Ruby</li>
                          <li class="skill-item">Html</li>
                          <li class="skill-item">Css</li>
                       </ul>
                    </li>
                    <li class="frameworks"> 
                      Frameworks
                      <ul>
                         <li class="skill-item">Bootstrap</li>
                         <li class="skill-item">Ruby on Rails</li>
                         <li class="skill-item">RSPec</li>
                         <li class="skill-item">CapyBara</li>
                         <li class="skill-item">Selenium</li>
                      </ul>
                    </li>
                    <li class="skills"> 
                       Skills
                       <ul>
                         <li class="skill-item">Codekit</li>
                        <li class="skill-item">Github</li>
                        <li class="skill-item">Codepen</li>
                        <li class="skill-item">Gitlab</li>
                        <li class="skill-item">Terminal</li>
                       </ul>
                    </li>
                </ul>
```